homeassistant/components/sensor/tibber.py

```diff
@@ -191,7 +191,11 @@ def __init__(self, tibber_home):
         if live_measurement is None:
             return
         self._state = live_measurement.pop('power', None)
-        self._device_state_attributes = live_measurement
+        for key, value in live_measurement.items():
+            if value is None:
```

Comment:
Should we label this a breaking change?

Reply:
No, the old attributes will be unchanged. 
But the new attributes will update more infrequent, and the streamed values will be None when they are missing. (All existing attributes are updated each time and never None)

Reply:
Just noting that old state attributes will remain, even if the new values are `None`. Maybe that's ok?

Reply:
Yes, that is the intention.